app/database.js

```diff
@@ -0,0 +1,16 @@
+const { MongoClient } = require('mongodb');
+
+const uri = 'mongodb://localhost:27017';
```

Comment:
Do not hard-code the credentials use environment variable I see a dotenv dependency why are you not using it